src/customFilters.js

```diff
@@ -0,0 +1,85 @@
+const Markdown = require('markdown-it');
+const _ = require('lodash');
+const OpenAPISampler = require('openapi-sampler');
+
+const filter = module.exports;
+
+/**
+Filters that go away by adding loadash:
+
+split
+isArray
+isObject
+firstKey -> head
+contains -> includes
+*/
+
+/* html-template only
+isExpandable
+containTags
+containNoTag
+nonParserExtensions
+*/
+
+/**
+ * Turns Markdown into HTML
+ * @md {string} - String with valid Markdown syntax
+ * @returns {string} HTML string
+ */
+function markdown2html(md) {
+  return Markdown().render(md || '');
+}
+filter.markdown2html = markdown2html;
```

Comment:
What about rewriting the filters like this:

```suggestion
filter.markdown2html = function(md) {
  return Markdown().render(md || '');
}
```

Reply:
yeah, but the problem is that once I do it this way `jsdoc2md` doesn't include this function in the docs

Reply:
Oh got it! Forget about it then 👍